services/SubmissionService.ts

```diff
@@ -36,6 +37,27 @@ export const getSubmissions = async (currentPage: number, status: string) => {
   return res.data;
 };
 
+export const getOwnSubmissions = async (
+  userId: number | undefined,
+  currentPage: number,
+  status: Status | ""
+) => {
+  const res = await axios.get(
+    `http://localhost/api/submission/user/
```

Comment:
the base url should be put inside a dev env file instead of being hardcoded if you need to change it depending on ... ya know, the environment